pytket/qir/conversion/conversion.py

```diff
@@ -422,14 +422,7 @@ def _rebase_op_to_gateset(self, op: Op, args: list) -> Optional[Circuit]:
     def _get_optype_and_params(self, op: Op) -> tuple[OpType, Sequence[float]]:
         optype: OpType = op.type
         params: list = []
-        if optype == OpType.ExplicitPredicate:
```

Comment:
I am not sure sure why this was in here, I was note able to construct a case where this was used. Everything is still working without this.